src/main/groovy/grails/plugin/sentry/SentryGrailsPlugin.groovy

```diff
@@ -59,8 +60,10 @@ class SentryGrailsPlugin extends Plugin {
 
             if (pluginConfig?.dsn) {
                 log.info 'Sentry config found, creating Sentry client and corresponding Logback appender'
-                grailsSentryClientFactory(GrailsSentryClientFactory)
-                sentryClient(grailsSentryClientFactory: 'createSentryClient', pluginConfig.dsn.toString()) { bean ->
+                sentryClientFactoryProvider(SentryClientFactoryProvider)
```

Comment:
v looks much better. I would be still a bit afraid of breaking backwards compatibility by removing the `grailsSentryClientFactory` completely

Reply:
Yes, i think that giving back to the bean its initial name `sentryFactory` will preserve the compatibility, WDYT ?

Reply:
yes!